satpy/tests/reader_tests/test_satpy_cf_nc.py

```diff
@@ -33,6 +33,60 @@
 # - tmp_path
 
 
+def _create_test_netcdf(filename, resolution=742):
+    res = 2
+    if resolution == 371:
+        res = 4
+    data_visir = np.array(np.arange(1, res * res + 1)).reshape(res, res)
+    lat = 33.0 * data_visir
+    lon = -13.0 * data_visir
+
+    lat = xr.DataArray(lat,
+                       dims=('y', 'x'),
+                       attrs={'name': 'lat',
+                              'standard_name': 'latitude',
+                              'modifiers': np.array([])})
+    lon = xr.DataArray(lon,
+                       dims=('y', 'x'),
+                       attrs={'name': 'lon',
+                              'standard_name': 'longitude',
+                              'modifiers': np.array([])})
+
+    solar_zenith_angle_i = xr.DataArray(data_visir,
+                                        dims=('y', 'x'),
+                                        attrs={'name': 'solar_zenith_angle',
+                                               'coordinates': 'lat lon',
+                                               'resolution': resolution})
+
+    scene = Scene()
+    scene.attrs['sensor'] = ['avhrr-1', 'avhrr-2', 'avhrr-3']
```

Comment:
Looks like the resolution indicates more viirs data, should we have viirs here for consistency/realism?